app/javascript/react/components/WeekView/WeeklyMockData.tsx

```diff
@@ -0,0 +1,27 @@
+const today = new Date();
+
+const getDayAhead = (ahead: number): Date => {
+  const today = new Date();
+  const upDate = today.setDate(today.getDate() + ahead);
+  return new Date(upDate);
+};
+
+const weeklyData = [
+  { value: 25, date: today },
+  { value: 90, date: getDayAhead(1) },
+  { value: 80, date: getDayAhead(2) },
+  { value: 2, date: getDayAhead(3) },
+  { value: 60, date: getDayAhead(4) },
+  { value: 200, date: getDayAhead(5) },
+  { value: 30, date: getDayAhead(6) },
+];
```

Comment:
I know it's mocked but to be sure that you know. It should be the other way around. It's `today` and 6 previous days. Unless it works different on week view but I'm 99% sure it's not.